CHANGES.md

```diff
@@ -30,6 +30,10 @@ OpenSSL 3.5
 
 ### Changes between 3.4 and 3.5 [xx XXX xxxx]
 
+* Added server side support for QUIC
+
+   *Matt Caswell, Tomáš Mráz, Neil Horman, Sasha Nedvedicky, Andrew Dinh*
```

Comment:
Shouldn't Hugo Landau be listed here? Around ~100/280 commits in #26762 are from him.

Reply:
> Shouldn't Hugo Landau be listed here? Around ~100/280 commits in #26762 are from him.

Absolutely - that should be fixed. Hugo did an amazing amount of work on that feature.

Reply:
Indeed. Hugo should probably be listed first IMO.